Feature/translation fixes

harshvitra opened the pull request:
Fixes #1672

Except for the error messages in create competition

@norbertschuler

harshvitra commented:
> I am a little concerned about the usage of `toLocaleString` instead of `delimitNumbers` for our number formatting, but hopes it all works correctly:

I have removed` toLocaleString` and added `delimitNumbers`, where else do we have `toLocaleString`

norbertschuler commented:
> I have removed` toLocaleString` and added `delimitNumbers`, where else do we have `toLocaleString`

My above screen shot is the result of searching the whole `/app` folder, so I think if you changed all these we do not used `toLocaleString` any more. You have not yet pushed your changes to GitHub.

harshvitra commented:
> My above screen shot is the result of searching the whole `/app` folder, so I think if you changed all these we do not used `toLocaleString` any more, but you have not yet pushed your changes to GitHub, have you @harshvitra ?

I did the same search and replaced it, dont know where it got lost, added again
